doc/adr/0004-nats-headers.md

```diff
@@ -82,7 +95,8 @@ HMSG <SUBJECT> <SID> [REPLY] <HDR_LEN> <TOT_LEN>
 <PAYLOAD>
 ```
 
-- `HDR_LEN` includes the entire serialized header, from the start of the version string up to and including the ␍␊ before the payload
+- `HDR_LEN` includes the entire serialized header, from the start of the version
```

Comment:
Again, not sure why we mention version here? Are you referring to `NATS/1.0\r\n`?